docs/_manage/versions/versions.md

```diff
@@ -40,6 +40,12 @@ Learn more on:
 
 Integrations created with the Platform CLI cannot be edited in the Platform UI, however you can view the available versions in the Platform UI. You can also run the [`zapier versions`](https://github.com/zapier/zapier-platform/blob/master/packages/cli/docs/cli.md#versions) command to see the same information in your local terminal.
 
+### What do I do if I am blocked from pushing new versions?
+
+Zapier may fix bugs or add new features to your integration and release a new version. You can run `zapier pull` to update your local files with the latest version.
+
+Any destructive file changes will prompt you with a confirmation dialog before continuing.
+
```

Comment:
Should include something like:

In the event that Zapier makes changes to your integration, you will be unable to do the following until you bring down the changes until you run `zapier pull`. 
- pushing to the promoted version
- promoting a new version
- migrating from one version to another version

Reply:
Good thinking - I meshed those things in 👍